frontend/src/global_styles/content/_autocomplete.sass

```diff
@@ -137,7 +148,9 @@ div.autocomplete
     margin-right: 8px
 
   &.ng-option-marked
-    background-color: $spot-color-basic-gray-6 !important
+    background-color: var(--control-transparent-bgColor-hover) !important
```

Comment:
I saw that pattern quite often now of using the control colours for something that is no control element. Could you please check whether there is an alternative?